cmd/search.go

```diff
@@ -141,6 +150,8 @@ func fuzzySearchResults(results spotify.SearchResult, searchType string) spotify
 		return results.Albums.Albums[idx].URI
 	case "playlist":
 		return results.Playlists.Playlists[idx].URI
+	case "artist":
+		return results.Artists.Artists[idx].URI
```

Comment:
I thiiiiiink there's more that needs to be returned here, I'm not entirely sure, as the behavior should be playing the top tracks from an artist, then playing more songs from that artist. Check how the `tracks` case works for the latter part of this.

Reply:
Actually, I lied, this is fine. This change goes elsewhere.